src/stories/crossingListItem.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { MemoryRouter } from 'react-router';
```

Comment:
Hooray for good tests!

Reply:
Haha too bad this doesn't really confirm that the tests are good or bad. 😉